### PR TITLE
Add template for GitHub issue forms

### PR DESCRIPTION
**Pull request by pascalberger**
Add template for GItHub issue forms

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ask for help or share ideas
+    url: https://github.com/cake-build/cake/discussions//category_choices
```
Comment (augustoproiete) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    url: https://github.com/cake-build/cake/discussions/category_choices
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Operating system
+    options:
+      - Linux
+      - Windows
+      - MacOS
+  validations:
```
Comment (augustoproiete) [inline .github/ISSUE_TEMPLATE/bug-report.yml:34]: - Can/Should this be a multi-select option? e.g. Bugs affect Linux and MacOS, but not Windows.
- Should we have an `N/A` option? There are cases where the OS is not relevant.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    required: true
+- type: dropdown
+  attributes:
+    label: Operating system architecture
+    options:
+      - 32-Bit
+      - 64-Bit
+  validations:
```
Comment (augustoproiete) [inline .github/ISSUE_TEMPLATE/bug-report.yml:42]: - Should we have an `N/A` option? There are cases where the OS architecture is not relevant.

**Review by augustoproiete — APPROVED** [review]
LGTM. A couple of suggestions added in comments

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 3 above)
Comment (pascalberger) [reply .github/ISSUE_TEMPLATE/bug-report.yml:46]: Added `N/A` option

**Pull request merged**